jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -830,15 +830,18 @@ protected final Class<?> findSystemClass (String className) throws ClassNotFound
  *
  * @return 		java.lang.ClassLoader
  *					the class or null.
+/*[IF JAVA_SPEC_VERSION < 24]
  * @exception	SecurityException
  *					if a security manager exists and it does not
  *					allow the parent loader to be retrieved.
+/*[ENDIF] JAVA_SPEC_VERSION < 24
  */
 @CallerSensitive
 public final ClassLoader getParent() {
 	if (parent == null) {
 		return null;
 	}
```

Comment:
This can move into the pre-Java24 block; either way line 855 will do the right thing.